pkg/machineproviders/providers/openshift/machine/v1beta1/provider.go

```diff
@@ -117,6 +153,12 @@ type openshiftMachineProvider struct {
 
 	// providerConfig stores the providerConfig for creating new Machines.
 	providerConfig providerconfig.ProviderConfig
+
+	// namespace store the namespace where new machines will be created
+	namespace string
+
+	// machineAPIScheme contains scheme for Machine API v1 and v1beta1
+	machineAPIScheme *apimachineryruntime.Scheme
```

Comment:
Nit, comments should end in periods
```suggestion
	// namespace store the namespace where new machines will be created.
	namespace string

	// machineAPIScheme contains scheme for Machine API v1 and v1beta1.
	machineAPIScheme *apimachineryruntime.Scheme
```